### DISCUSSION TITLE
FATAL: REPOSITORY....IS Owned by SOMEONE ELSE

### DISCUSSION THREAD

**Discussion by MikeB76** [community/community#28371] [Other Feature Feedback, Questions, & Ideas]
I have Committed and Pushed to several Repositories for several years until this past April when either an upgrade to 2.36, or some other event started the following Warnings and Fatal errors, below, with my directories. What to do? I have gone back to 2.35 and advanced to 2.37. Same results.

L:\BAT>git status
warning: '//10.12.200.8/rct/Development/BAT' is owned by:
        'S-1-5-21-1757981266-583907252-1801674531-38077'
but the current user is:
        'S-1-12-1-3784088403-1157673933-3654986916-1184536155'
fatal: unsafe repository ('//10.12.200.8/rct/Development/BAT' is owned by someone else)
To add an exception for this directory, call:

        git config --global --add safe.directory '%(prefix)///10.12.200.8/rct/Development/BAT'

L:\BAT>git config --global --add safe.directory '//10.12.200.8/rct/Development/BAT'
warning: '//10.12.200.8/rct/Development/BAT' is owned by:
        'S-1-5-21-1757981266-583907252-1801674531-38077'
but the current user is:
        'S-1-12-1-3784088403-1157673933-3654986916-1184536155'

**Comment by airtower-luna** [comment]
I notice the command you ran isn't *exactly* the same a the one suggested by `git status`, I assume that's why it's not working: the `%(prefix)/` part at the start of the repository path is missing.

However, before repeating the command check [the documentation on `safe.directory`](https://git-scm.com/docs/git-config/#Documentation/git-config.txt-safedirectory) to understand what it does. [This blog post](https://github.blog/2022-04-12-git-security-vulnerability-announced/) explains *why* those limitations (and the opt-out) were introduced.
